### ISSUE TITLE
Figure out classes that need to be renamed for migration script

### ISSUE THREAD

**Issue by mdedetrich** [apache/incubator-pekko#68]
In https://github.com/apache/incubator-pekko/pull/67 we renamed all of the classes that contain `Akka` in the name to `Pekko`. We should prune these renamed classes and figure out which ones are relevant for the migration script.

**Comment by sam-byng** [comment]
Following discussion in #22 , Could we label as 'nice to have (low prio)' ?

**Comment by mdedetrich** [comment]
Not entirely sure if this is low priority or not, my initial assumption is that this would in fact be very useful and hence somewhat critical for people migrating massive akka to pekko codebases.

It is true that this will be a bit of work however there is already prior art for using scala-migrate in pekko http

**Comment by He-Pin** [comment]
But I think that part can come up later, a 1.0.0 release can be used for newly started projects.

**Comment by mdedetrich** [comment]
True. I also just realized that while scala-migrate works for Scala projects, it doesn't work for Java and a lot of people do use akka in Java projects.

**Comment by pjfanning** [comment]
removed from 1.0.0 milestone - it might be useful just to close it (@mdedetrich) - we have https://github.com/apache/incubator-pekko/issues/22 already
